awesome_owl/static/src/card.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates xml:space="preserve">
+    <t t-name="awesome_owl.card">
+        <div class="card d-inline-block m-2" style="width: 18rem;" t-on-click="toggleCard">
+            <div class="card-body">
+                <h5 class="card-title"><t t-esc="props.title"/></h5>
+                <t t-if="state.opened">
+                    <t t-slot="default"/>
+                </t>
+            </div>
+        </div> 
+        readonly="state in ('offer_accepted
```

Comment:
missing the end here I believe :wink:

Reply:
I think I just made a mistake it shouldn't be here